Description: Removes argus-pap from the LSB header Required-Start and Required-Stop.
Author: Valery Tschopp <valery.tschopp@switch.ch>
--- a/src/main/scripts/init.d/argus-pdp
+++ b/src/main/scripts/init.d/argus-pdp
@@ -23,8 +23,8 @@
 #
 ### BEGIN INIT INFO
 # Provides:          argus-pdp
-# Required-Start:    $network $remote_fs argus-pap
-# Required-Stop:     $network $remote_fs argus-pap
+# Required-Start:    $network $remote_fs
+# Required-Stop:     $network $remote_fs
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
 # Short-Description: Start pdp at boot time
